.github/workflows/dependency_changelog.yml

```diff
@@ -0,0 +1,111 @@
+# `dependencies` label at any time.
+
+name: Dependency Changelog
+
+on:
+  pull_request:
+    # catch when the PR is opened with the label or when the label is added
+    types: [opened, labeled]
+
+permissions:
+  contents: write
+  pull-requests: read
+
+jobs:
+  dependency_changelog:
+    if: ${{ github.event.label.name == 'dependencies' }}
+    runs-on: ubuntu-latest
+
+    steps:
+    # timestamp changes the order the changelog entries are listed in the final Changelog.md file.  Precision is not
+    # important here.
+    # The timestamp on the filename and the timestamp in the contents of the file have different expected formats.
+    - name: Get File Name Timestamp
+      id: filename_time
+      uses: nanzm/get-time-action@v1.1
+      with:
+        format: 'YYYYMMDD-HHmmss'
+    - name: Get File Content Timestamp
+      id: file_content_time
+      uses: nanzm/get-time-action@v1.1
+      with:
+        format: 'YYYY-MM-DDTHH:mm:ss.000000-05:00'
+    # changie expects files to be named in a specific pattern.
+    - name: Generate Filepath
+      id: fp
+      run: |
+        FILEPATH=.changes/unreleased/Dependencies-${{ steps.filename_time.outputs.time }}.yaml
+        echo "::set-output name=FILEPATH::$FILEPATH"
+    - name: Check if changelog file exists already
+      # if there's already a changelog entry, don't add another one!
+      # https://github.com/marketplace/actions/paths-changes-filter
+      # For each filter, it sets output variable named by the filter to the text:
+      #  'true' - if any of changed files matches any of filter rules
+      #  'false' - if none of changed files matches any of filter rules
+      # also, returns:
+      #  `changes` - JSON array with names of all filters matching any of the changed files
+      uses: dorny/paths-filter@v2
+      id: filter
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        filters: |
+          changelog:
+            - added: '.changes/unreleased/**.yaml'
+    - name: Checkout Branch
+      if: steps.filter.outputs.changelog == 'false'
+      uses: actions/checkout@v2
+      with:
+        # persist-credentials: false is crucial otherwise Git push is performed with
+        # github.token and not the token you configure using the env: GITHUB_TOKEN.
+        # See: https://github.com/gr2m/create-or-update-pull-request-action/issues/281
+        persist-credentials: false
+    - name: Get name and email for commit
+      # The action to update the PR expects the author to be in the format of name <email>.
+      # Grab the name and email from the last commit to this PR and use it to commit the new
+      # changelog file
+      if: steps.filter.outputs.changelog == 'false'
+      id: user
+      run: |
+        USERNAME="$(git log -n 1 --pretty=format:%an)"
+        USEREMAIL="$(git log -n 1 --pretty=format:%ae)"
+        echo "::set-output name=USERNAME::$USERNAME"
+        echo "::set-output name=USEREMAIL::$USEREMAIL"
```

Comment:
Would _love_ to hear this is not the best way to get the name/email but it's the best I could come up with!

Reply:
@leahwicz  it makes sense to replace this with a bot as we discussed offline. [dbtBot](https://github.com/dbt-labs/dbtBot) is the only one I could find but it doesn't seem right by the README?

Reply:
There's also @FishtownBuildBot

Reply:
Let's use https://github.com/FishtownBuildBot. We use it for some commits for version bumping now that I look
https://github.com/dbt-labs/dbt-core/blob/080dd41876aaf5d348f75db402b2efbb074be4a2/.github/workflows/version-bump.yml#L95